static/css/legacy-tools.less

```diff
@@ -141,21 +141,16 @@
       margin-left: 0;
```

Comment:
I think all this CSS is now dead; this applies to `table.buy-options-table`, which no longer exists (since it's a `ul`).

Reply:
done